Makefile.am

```diff
@@ -212,6 +212,7 @@ AM_JS_LOG_FLAGS = --tap --no-config @JASMINE_REPORT_ARGUMENT@
 TESTS_ENVIRONMENT = \
 	export GJS_PATH="$(top_srcdir):$(top_srcdir)/lib:$(top_builddir):$(top_builddir)/lib"; \
 	export GI_TYPELIB_PATH="$(top_builddir)$${GI_TYPELIB_PATH:+:$$GI_TYPELIB_PATH}"; \
+	export GSETTINGS_SCHEMA_DIR="/var/run/host/usr/share/glib-2.0/schemas"; \
```

Comment:
Note: `GSETTING_SCHEMA_DIR` can only have one value. I'm using that instead of `XDG_DATA_DIRS` since I wasnt sure if it was right to expose all of the host's data dirs to the service.